camayoc/tests/qcs/api/v1/scans/test_satellite_scans.py

```diff
@@ -0,0 +1,167 @@
+# coding=utf-8
+"""Tests for ``Scan`` API endpoint for quipucords server.
+
+:caseautomation: automated
+:casecomponent: api
+:caseimportance: high
+:caselevel: integration
+:requirement: Sonar
+:testtype: functional
+:upstream: yes
+"""
+
+import pytest
+
+from camayoc import config
+from camayoc.exceptions import ConfigFileNotFoundError
+from camayoc.qcs_models import (
+    Credential,
+    Source,
+    Scan
+)
+from camayoc.tests.qcs.api.v1.scans.utils import wait_until_state
+
+
+TIMEOUT = 540
```

Comment:
Docstring?

Reply:
I see this is the timeout for the scan, maybe naming as SCAN_TIMEOUT would help avoid naming conflict and also give more clarity. But this is up to you, the docstring would help clarify too.